backend/src/main/java/com/soen341/instagram/controller/ProfileController.java

```diff
@@ -1,92 +1,82 @@
 package com.soen341.instagram.controller;
 
+import org.modelmapper.ModelMapper;
 // Spring Boot
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
 // Project
 import com.soen341.instagram.dto.account.ProfileRequestDTO;
+import com.soen341.instagram.model.Account;
 import com.soen341.instagram.service.impl.ProfileService;
 
 @RestController
-public class ProfileController {
-	
+public class ProfileController
+{
+
 	@Autowired
 	private ProfileService profileService;
-	
+	@Autowired
+	private ModelMapper modelMapper;
+
 // For now we are not allowing the user to change their username
```

Comment:
Is this something we think we'll (realistically) have time to implement before the end of the sprint? If not I'd advise we remove it

Reply:
I don't think this is something we will have time to implement before the end of the sprint and, furthermore, I don't think this is something we should implement. It would require massive backend changes for very little payoff. Rather than allowing people to change their username, we can simply show their display name on all of their comments/posts since it would have a similar effect.

Reply:
We can remove it, but create another issue for it